### PR TITLE
units: add page

### PR DESCRIPTION
**Pull request by ryangreenberg**
- [x] The page (if new), does not already exist in the repo.
- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.
- [x] The page has 8 or fewer examples.
- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.
- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/units.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+- Show the conversion between two simple units:
+
+`units quarts tablespoons`
+
+- Show the conversion between two compound units:
+
+`units "meters / second" "inches / hour"`
+
+- Run `units` in interactive mode:
```
Comment (agnivade) [inline pages/common/units.md]: This should be the first example.

File: pages/common/units.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# units
+
+> Provide the conversion between two units of measure.
+
+- Show the conversion between two simple units:
+
+`units quarts tablespoons`
```
Comment (agnivade) [inline pages/common/units.md]: Values should be inside tokens (`{{}}`).

File: pages/common/units.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+- Show the conversion between two compound units:
+
+`units "meters / second" "inches / hour"`
```
Comment (agnivade) [inline pages/common/units.md]: Same as above.

File: pages/common/units.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# units
+
+> Provide the conversion between two units of measure.
+
```
Comment (agnivade) [inline pages/common/units.md]: Add this line in a sub-heading-  `Typing 'search {{text}}' in the prompt  will display a list of all of the units containing {{text}}.`

Comment (ryangreenberg) [reply pages/common/units.md]: Looks like that is the behavior of GNU units but not the units that is installed by default on, e.g. OS X. How do you want to handle the difference?

**Comment by ryangreenberg** [comment]
@agnivade Thank you for the feedback! I added a new example that included quantities (to help people understand they can use `units` with or without them).

File: pages/common/units.md (hunk 4 above)
Comment (agnivade) [reply pages/common/units.md]: Ah, in that case, you need to create separate copies of the page inside linux, windows and osx folders.

**Comment by agnivade** [comment]
We need to take care of this https://github.com/tldr-pages/tldr/pull/2038#discussion_r175869313

**Comment by ryangreenberg** [comment]
@agnivade I added your suggested search message to the linux page. I don't know what (if anything) the windows version supports so I left it the same as the osx page.

**Pull request merged**